legend-pure-m2-functions-pure/src/main/resources/platform_functions/meta/newQualifiedProperty.pure

```diff
@@ -0,0 +1,15 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+native function meta::pure::functions::meta::newQualifiedProperty(name:String[1], owner:GenericType[1], target:GenericType[1], multiplicity:Multiplicity[1], functionType:FunctionType[1]):QualifiedProperty<Any>[1];
```

Comment:
I don't think this should take the function type as an argument. Instead, I think you should should take the parameters as an argument. The function type should be assembled inside the native implementation from the owner, target, multiplicity, and parameters.

Reply:
Makes complete sense...that was my initial feeling..but wanted to confirm with you as this approach is slightly harder. Thanks